tests/Nelmio/Alice/Fixtures/Builder/Reference.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *  
+ * (c) Nelmio <hello@nelm.io>
+ *  
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Nelmio\Alice\Fixtures\Builder\Methods;
+
+use Nelmio\Alice\Fixtures\Fixture;
+
+/**
+ * List a list of fixture reference, with the expected result. The list is flagged to be able to easily extract some
+ * parts of it. This class aims at centralising all the possible fixture tested to avoid too much duplication.
+ */
+final class Reference
```

Comment:
>I'm personally not in favor of using engineered solutions in tests (factories, etc.) since it makes things a bit more opaque

I feel the same, duplication is not bad in tests. However in this case doing it feels... overwhelming. The idea was to get a complete list of references somewhere which can be used and re-used. Sometimes the whole thing (for integration tests), sometimes just some bits (for some unit tests). The idea is that unlike the integration and unit tests, they will be portable to v3 as well.